### PR TITLE
[feat] Add clear button to clear all the programme script

### PR DESCRIPTION
**Pull request by emettely**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      

**Describe what the PR does**    
1. new button to clear-all
Screenshot 2021-03-31 at 14 46 51
2. prompts to confirm
Screenshot 2021-03-31 at 14 46 54
3. clears and updates database
Screenshot 2021-03-31 at 14 46 58

**State whether the PR is ready for review or whether it needs extra work**    
ready

**Additional context**    
This is something that I had in one of my local branches 😅 making sure they don't get lost...

### CODE REVIEW

**Review by allishultes — APPROVED** [review]
Tested this locally and looks good to me, Eimi!

**Comment by emettely** [comment]
This needs another review before it can be merged

File: src/Components/PaperEditor/ProgrammeScriptContainer/index.js [hunk 1]
```diff
@@ -90,6 +90,23 @@ const ProgrammeScriptContainer = (props) => {
...
+        {
+          type: 'insert',
+          text: 'Insert point to add selection',
+        }
+      ];
+      setElements(newList);
+      setResetPreview(true);
+
+      handleSaveProgrammeScript(newList);
+      props.trackEvent({ category: 'paperEditor programmeScript', action: 'handleDelete all elements' });
```
Comment (tamsingreen) [inline src/Components/PaperEditor/ProgrammeScriptContainer/index.js:109]: Should the action be called `handleClearProgrammeScript` as the others use this convention?

**Review by tamsingreen — COMMENTED** [review]
Looks great!

File: src/Components/PaperEditor/ProgrammeScriptContainer/index.js (hunk 1 above)
Comment (emettely) [reply src/Components/PaperEditor/ProgrammeScriptContainer/index.js:109]: Thanks!

**Pull request merged**